setup.cfg

```diff
@@ -140,6 +140,7 @@ install_requires =
     python-nvd3>=0.15.0
     python-slugify>=5.0
     rich>=12.4.4
+    rich-click>=1.3.1
```

Comment:
One small thing (for next PR). Rich-click 1.5 introduced some changes that might cause incompatibilities. I tihnk bumping up to > 1.5 is a good idea.